shoulder/__main__.py

```diff
@@ -26,12 +26,30 @@
 from shoulder.logger import logger
 from shoulder.parser import *
 from shoulder.generator import *
-from shoulder.filters import *
+from shoulder.filter import filters
+from shoulder.transform import transforms
 
 regs = parse_registers(config.xml_register_dir)
 logger.debug("Registers parsed: " + str(len(regs)))
 
-regs = apply_filters(regs)
-logger.debug("Registers after filters: " + str(len(regs)))
+regs = filters["activity_monitor"].filter_exclusive(regs)
```

Comment:
Since we are applying all the filters here, maybe it would be more compact to do something like:
```
for filter in filters:
    regs = filter.filter_exclusive(regs)
```